### DISCUSSION TITLE
v0.2.1 Beta

### DISCUSSION THREAD

**Discussion by yoshiask** [yoshiask/FluentStore#22] [Announcements]
## What's new
- Fixed "400 Bad Request" error when attempting to load details for MS Store apps
- Fixed file extension when downloading certain apps
- Added support for unpackaged apps in the MS Store
- Added UWP Community apps
- Success and error messages are now shown as banners at the bottom of the window
- Clicking on a screenshot expands it to the full window
- You can now select multiple lines in the "Features" list on an app page
- Added a download cache

**Full Changelog**: https://github.com/yoshiask/FluentStore/compare/v0.1.2-beta...v0.2.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/yoshiask/FluentStore/releases/tag/v0.2.1-beta'>v0.2.1 Beta</a>.</em>

**Comment by TheNonSmoker** [comment]
Have you abandoned this project?

**Reply by yoshiask** [reply]
I'm a student working on this in my rare free time, but no, I haven't abandoned it. If you'd like to see what kind of things are being working on, you can check the most recent commits. Thanks for checking it out!

**Reply by TheNonSmoker** [reply]
Thanks, I'll check :)
